rental_pricelist/tests/test_rental_pricelist.py

```diff
@@ -177,13 +186,13 @@ def test_01_rental_onchange_productA(self):
             )
         )
         line.onchange_display_product_id()
-        res = line.product_id_change()
+        line.product_id_change()
         # check uom domain
-        check_uom_domain = False
-        if "domain" in res and "product_uom" in res["domain"]:
-            self.assertEqual(len(res["domain"]["product_uom"][0][2]), 3)
-            check_uom_domain = True
-        self.assertTrue(check_uom_domain)
+        # check_uom_domain = False
```

Comment:
Why do you deactivate this check?

Reply:
in v14 `defproduct_id_change()` in model `sale.order.line`, we do not have "domain" as key in retrun `res`

Reply:
Then we can adjust the if clause in product_id_change() and add the key domain.